### DISCUSSION TITLE
I can't create a project in Quasar.  installation problems on windows

### DISCUSSION THREAD

**Discussion by Woodman06** [quasarframework/quasar#10602] [Quasar App Extensions] [answered]
When I run the command: "npx quasar create my-project". Gives an error: command not found quasar. 
Can you help me figure out what the error is?
![i5N3GPdXiWQ](https://user-images.githubusercontent.com/90316569/132487621-91f8d865-77dd-4134-ada8-93e6399a9c59.jpg)

**Comment by MilosPaunovic** [comment, marked as answer]
You don't have Quasar installed.
Follow this instruction: https://quasar.dev/quasar-cli/installation#introduction

**Reply by Woodman06** [reply]
how can it be removed and installed in the folder that is written?
